keywords/couchbaseserver.py

```diff
@@ -81,7 +83,7 @@ def create_internal_rbac_bucket_user(url, bucketname):
         resp.raise_for_status()
     except HTTPError as h:
         log_info("resp code: {}; error: {}".format(resp, h))
-        raise
+        raise RBACUserCreationError(h)
```

Comment:
A simple string would be fine as a parameter, i.e.

`raise RBACUserCreationError('Could not create RBAC user!')`